dropwizard-client/src/test/java/io/dropwizard/client/JerseyClientIntegrationTest.java

```diff
@@ -0,0 +1,268 @@
+package io.dropwizard.client;
+
+import com.codahale.metrics.MetricRegistry;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.base.Charsets;
+import com.google.common.base.Optional;
+import com.google.common.io.CharStreams;
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import io.dropwizard.jackson.Jackson;
+import org.junit.*;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.*;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.util.concurrent.Executors;
+import java.util.zip.Deflater;
+import java.util.zip.GZIPInputStream;
+import java.util.zip.GZIPOutputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Integration test of {@link org.glassfish.jersey.client.JerseyClient}
+ * with {@link io.dropwizard.client.DropwizardApacheConnector}
+ */
+public class JerseyClientIntegrationTest {
+
+    private static final String TRANSFER_ENCODING = "Transfer-Encoding";
+    private static final ObjectMapper JSON_MAPPER = Jackson.newObjectMapper();
+    private static final String APPLICATION_JSON = "application/json";
```

Comment:
Can be replaced by a static import of `javax.ws.rs.core.MediaType.APPLICATION_JSON`.